projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/initVDKConfigCell.ts

```diff
@@ -11,34 +11,74 @@ import { INotebookTracker } from '@jupyterlab/notebook';
 export function initVDKConfigCell(notebookTracker: INotebookTracker) {
   const notebookPanel = notebookTracker.currentWidget;
   if (notebookPanel) {
-    const initCell =
+    // Create Markdown Cell
+    const markdownCell =
+      notebookPanel.content.model?.contentFactory?.createMarkdownCell({
+        // eslint-disable-next-line prettier/prettier
+        cell: {
+          cell_type: 'markdown',
+          source: [
+            'You are running in environment where VDK Jupyter extension is installed.<br/>',
+            'If you are not using VDK, you can delete and ignore this and below cell.<br/><br/>',
+            'To learn more check out [VDK Notebook Getting Started](https://bit.ly/vdk-notebook).<br/><br/>',
+            '**IMPORTANT: Please execute the cell below to load the VDK job_input variable.**'
+          ],
+          metadata: { editable: false }
+        }
+      });
+
+    // Create Code Cell
+    const codeCell =
       notebookPanel.content.model?.contentFactory?.createCodeCell({
         cell: {
           cell_type: 'code',
           source: [
-            `"""\n`,
-            `This cell must be executed to load VDK job_input variable .\n\n`,
-            `vdk.plugin.ipython extension introduces a magic command for Jupyter.\n`,
-            `The command enables the user to load VDK for the current notebook.\n`,
-            `VDK provides the job_input API, which has methods for:\n`,
-            `    * executing queries to an OLAP database;\n`,
-            `    * ingesting data into a database;\n`,
-            `    * processing data into a database.\n`,
-            `Type help(job_input) to see its documentation.\n`,
-            `"""\n`,
-            `%reload_ext vdk.plugin.ipython\n`,
-            `%reload_VDK\n`,
-            `job_input = VDK.get_initialized_job_input()`
+            'vdk.plugin.ipython extension introduces a magic command for Jupyter.\n',
```

Comment:
When i try to run this cell for me it says:
vdk.plugin.ipython extension introduces a magic command for Jupyter.
                       ^
SyntaxError: invalid syntax

It is still a python cell shouldn't we have the text commented?